client/js/connectionsAPI.js

```diff
@@ -87,56 +88,98 @@ var connectionsAPI = (function() {
+      var organizations = userInfo.organizations;
+      if (organizations.length > 0) {
+        var myOrganizationsLi = document.createElement('LI');
+        myOrganizationsLi.className = 'dropdown';
 
-        var myRestaurantsA = document.createElement('A');
-        myRestaurantsA.id = 'myRestaurantsButtonLink';
-        myRestaurantsA.className = 'dropdown-toggle';
-        myRestaurantsA.setAttribute('data-toggle', 'dropdown');
-        myRestaurantsA.setAttribute('role', 'button');
-        myRestaurantsA.href = '#';
-        myRestaurantsA.textContent = 'My restaurants';
+        var myOrganizationsA = document.createElement('A');
+        myOrganizationsA.id = 'myOrganizationsButtonLink';
+        myOrganizationsA.className = 'dropdown-toggle';
+        myOrganizationsA.setAttribute('data-toggle', 'dropdown');
+        myOrganizationsA.setAttribute('role', 'button');
+        myOrganizationsA.href = '#';
+        myOrganizationsA.textContent = 'My organizations';
 
         var caret = document.createElement('B');
         caret.className = 'caret';
 
-        myRestaurantsA.appendChild(caret);
-        myRestaurantsLi.appendChild(myRestaurantsA);
+        myOrganizationsA.appendChild(caret);
+        myOrganizationsLi.appendChild(myOrganizationsA);
 
         var organizationsMenu = document.createElement('UL');
-        organizationsMenu.className = 'dropdown-menu';
+        organizationsMenu.className = 'dropdown-menu multi-level';
         organizationsMenu.setAttribute('aria-labelledby',
-                                      'myRestaurantsButtonLink');
+                                      'myOrganizationsButtonLink');
         organizationsMenu.setAttribute('role', 'menu');
 
-        for (var index = 0; index < userInfo.organizations.length; index++) {
+        for (var index = 0; index < organizations.length; index++) {
+          if (! (is_organization_manager(organizations[index]) ||
+            hasRole(userInfo, rol.globalManager))) {
+            continue;
+          }
           var organizationLi = document.createElement('LI');
-          organizationLi.setAttribute('role', 'presentation');
+          organizationLi.className = 'dropdown-submenu';
 
           var organizationA = document.createElement('A');
-          organizationA.setAttribute('role', 'menuitem');
           organizationA.tabIndex = -1;
-          organizationA.href =
-            'myRestaurants.html?franchise=' +
-            userInfo.organizations[index].name;
+
+            organizationA.href = '#';
           organizationA.textContent = userInfo.organizations[index].name;
 
           organizationLi.appendChild(organizationA);
-          organizationsMenu.appendChild(organizationA);
+
+          //submenu
+          var organizationSubMenu = document.createElement('UL');
+          organizationSubMenu.className = 'dropdown-menu';
+
+          //restaurants submenu
+          var restaurantLi = document.createElement('LI');
```

Comment:
this code is duplicated three times, here and two additional times below

perhaps you can create a function for it